src/app/Library/CrudPanel/Traits/Fields.php

```diff
@@ -41,7 +41,7 @@ public function makeSureFieldHasNecessaryAttributes($field)
         if (isset($field['entity'])) {
             $field = $this->makeSureFieldHasRelationType($field);
             $field = $this->makeSureFieldHasModel($field);
-            // $field = $this->makeSureFieldNameMatchesRelation($field);
```

Comment:
Renamed `makeSureFieldNameMatchesRelation()` into `overwriteFieldNameFromEntity()`. Since that's what it does, overwrite something, that's what the name should say.